internal/proxy/task.go

```diff
@@ -265,7 +265,7 @@ func (cct *createCollectionTask) PreExecute(ctx context.Context) error {
 			return err
 		}
 		// validate vector field type parameters
-		if field.DataType == schemapb.DataType_FloatVector || field.DataType == schemapb.DataType_BinaryVector {
+		if field.DataType == schemapb.DataType_FloatVector || field.DataType == schemapb.DataType_BinaryVector || field.DataType == schemapb.DataType_Float16Vector {
```

Comment:
Add a util function to handle these checkers.

Reply:
Get